tests/validation/tests/v3_api/common.py

```diff
@@ -1805,15 +1805,20 @@ def validate_catalog_app(proj_client, app, external_id, answer=None):
     chart = chart_prefix + "-" + chart_suffix
     app_name = parameters[len(parameters) - 2].split("=")[1]
     workloads = proj_client.list_workload(namespaceId=ns).data
+    '''
```

Comment:
can you switch to use # for inline comments?

Reply:
Done